.pipelines/cni/pipeline.yaml

```diff
@@ -70,8 +70,6 @@ stages:
             artifact: npm_k8s
       - job:
         displayName: "Build k8s NPM Windows Test Suite Binary"
-        pool:

```

Comment:
why are we removing the pool name ?

Reply:
Because it is set at the stage level. This assigns every job below it the same pool